app/views/osbbs/show.html.slim

```diff
@@ -0,0 +1,23 @@
+.row 
+  .col-md-4
+    - flash.each do |type, msg|
+      div
+        = msg
+    .osbb_info
```

Comment:
OSBB info should be on the the same vertical line not some to the left some to the right